### DISCUSSION TITLE
wget to ftp server located in different pod is not working

### DISCUSSION THREAD

**Discussion by shaykeren** [linkerd/linkerd2#4609] [General]
I configure my environment with linkerd, and I found that I cannot call to ftp pod from a different pod, both of them are proxied with linkerd, 

Getting this error from wget : Error in server response. Closing. Giving up

**Comment by olix0r** [comment]
Hi @shaykeren. It's been a while since I've looked at the FTP protocol, but I believe that, due to its use of a secondary control connection, the protocol is server-speaks-first. So, you'll need to [skip the FTP ports](https://linkerd.io/2/features/protocol-detection/#configuring-protocol-detection) for now.  We're working on a way to better support these sorts connections, but they'll probably continue to require some annotation/configuration of some sort.

**Reply by shaykeren** [reply]
How can i skip this protocol?
Im using the k8s api for deployment. Can I set some annotations for it to skip?
